bin/client/src/interop/consolidate.rs

```diff
@@ -111,6 +111,12 @@ where
 
     // Ensure that the post-state matches the claimed post-state.
     if post_commitment != boot.claimed_post_state {
+        tracing::error!(
+            target: "client_interop",
+            "Post state validation failed. Claimed: {}, Actual: {}",
+            boot.claimed_post_state,
+            post_commitment
+        );
```

Comment:
```suggestion
        tracing::error!(
            target: "client_interop",
            claimed=%boot.claimed_post_state,
            actual=%post_commitment,
            "Post state validation failed.",
        );
```
let's use the kv format for logs